### ISSUE TITLE
'Link' tool seems to be broken?

### ISSUE THREAD

**Issue by michielbdejong** [SolidOS/solid-panes#81]
This is what I see when I try to create a Link
, using the version of databrowser that is included in NSS master:

<img width="502" alt="Screenshot 2019-05-09 09 13 49" src="https://user-images.githubusercontent.com/408412/57434219-c3ca5e80-723a-11e9-8747-7d2eb8b52427.png">

**Comment by michielbdejong** [comment]
@Vinnl bump

**Comment by Vinnl** [comment]
@timbl What is the Link pane supposed to do? The [`mintNew`](https://github.com/solid/solid-panes/blob/eeeca7e2ae55080c436be6384db0a5c925397083/links/linksPane.js#L27) function simply resolves without doing anything - is it simply unfinished? ([The commit message seems to imply so...](https://github.com/solid/solid-panes/commit/eeeca7e2ae55080c436be6384db0a5c925397083))

**Comment by timbl** [comment]
Yes it is unfinished.. the links pane was for a holding just  a set of links, like references to things discussed in a meeting, or relevant remote web sites to folder, etc ... it was a while ago
